internal/kernel/scheduler/scheduler.go

```diff
@@ -54,6 +55,7 @@ func (s *Scheduler) Tick(t int64, batchSize int) {
 
 	// dequeue cqes
 	for _, cqe := range s.aio.Dequeue(batchSize) {
+		util.Assert(cqe.Callback != nil, "cqe is nil")
```

Comment:
can we change the assert message to "cqe must not be nil" ? that way the message matches the condition and don't have to think backwards when reading it as documentation.

Reply:
also can we assert the other fields of cqe are not nil either. that way we are certain the loop was correct.